cinema/views.py

```diff
@@ -186,9 +264,37 @@ class OrderViewSet(
     authentication_classes = (TokenAuthentication,)
     permission_classes = (IsAuthenticated,)
 
+
+
     def get_queryset(self):
         return Order.objects.filter(user=self.request.user)
 
+
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "user",
+                type=OpenApiTypes.INT,
+                description="Id of the user."
+            )
+        ],
+        description="Or orders that belongs to the user withe the given id",
```

Comment:
There is a typo in the description: 'Or orders that belongs to the user withe the given id'. It should be 'Orders that belong to the user with the given id'.